commands/solenoid/SolenoidExtend.java

```diff
@@ -14,34 +14,12 @@
 	 * Command to set the state of a SolenoidSubsystem to
 	 * EXTEND(DoubleSolenoid.Value.kForward)
 	 * 
-	 * @param name   Name of Command
```

Comment:
don't modify solenoid commands in this branch

Reply:
see #245.